[2025-05-14T04:57:37.584+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ecommerce_data_ingestion.wait_for_olist_customers_dataset manual__2025-05-14T04:56:16+00:00 [queued]>
[2025-05-14T04:57:37.630+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ecommerce_data_ingestion.wait_for_olist_customers_dataset manual__2025-05-14T04:56:16+00:00 [queued]>
[2025-05-14T04:57:37.651+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-05-14T04:57:37.841+0000] {taskinstance.py:2191} INFO - Executing <Task(FileSensor): wait_for_olist_customers_dataset> on 2025-05-14 04:56:16+00:00
[2025-05-14T04:57:37.868+0000] {standard_task_runner.py:60} INFO - Started process 405 to run task
[2025-05-14T04:57:37.924+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'ecommerce_data_ingestion', 'wait_for_olist_customers_dataset', 'manual__2025-05-14T04:56:16+00:00', '--job-id', '39', '--raw', '--subdir', 'DAGS_FOLDER/ecommerce_etl_dag.py', '--cfg-path', '/tmp/tmpgkbhmcuc']
[2025-05-14T04:57:37.940+0000] {standard_task_runner.py:88} INFO - Job 39: Subtask wait_for_olist_customers_dataset
[2025-05-14T04:57:38.377+0000] {task_command.py:423} INFO - Running <TaskInstance: ecommerce_data_ingestion.wait_for_olist_customers_dataset manual__2025-05-14T04:56:16+00:00 [running]> on host 0a774477c13b
[2025-05-14T04:57:38.853+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ecommerce_data_ingestion' AIRFLOW_CTX_TASK_ID='wait_for_olist_customers_dataset' AIRFLOW_CTX_EXECUTION_DATE='2025-05-14T04:56:16+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-14T04:56:16+00:00'
[2025-05-14T04:57:38.935+0000] {taskinstance.py:2360} INFO - Skipping due to soft_fail is set to True.
[2025-05-14T04:57:38.970+0000] {taskinstance.py:1138} INFO - Marking task as SKIPPED. dag_id=ecommerce_data_ingestion, task_id=wait_for_olist_customers_dataset, execution_date=20250514T045616, start_date=20250514T045737, end_date=20250514T045738
[2025-05-14T04:57:39.132+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-14T04:57:39.527+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
